common/src/main/script/codegen.groovy

```diff
@@ -23,7 +23,7 @@ void convertTemplates(String templateDir,
     def keyName = keyPrimitive.capitalize()
     def replaceFrom = "(^.*)K([^.]+)\\.template\$"
     def replaceTo = "\\1" + keyName + "\\2.java"
-    def hashCodeFn = keyPrimitive.equals("long") ? "(int)(key ^ (key >>> 32))" : "(int) key"
+    def hashCodeFn = keyPrimitive.equals("long") ? "(int) (key ^ (key >>> 32))" : "(int) key"
```

Comment:
Agreed...I copy/pasted this code for the `PlatformDependent0.hashCode(...)` and got a style compile error.  I guess we wouldn't catch that with this code because it generating code.  That being said I could pull this out...or just leave it and it will get fixed when I merge forward.